README.md

```diff
@@ -29,6 +29,14 @@ Drag mouse around the screen to create fluid particles with velocities in
 direction of mouse travel. Fluid fades slowly over time so as not to fill
 the container.
 
+### Parallel Inclusive Scan
+*** This example is work in progress so may not work *** \
```

Comment:
I've given both examples a try and they appear to be working using oneAPI 2023.2, there are some deprecation warnings (which we should fix separately) but I don't think we expect any failures in compilation or running

Reply:
It was discussed that just copying the code across was sufficient for now and a new task submitted to would ensure the examples worked.

Reply:
The warning text has been removed for both new examples. 
Done.